src/mpid/ch4/netmod/ofi/init_addrxchg.c

```diff
@@ -160,7 +160,7 @@ int MPIDI_OFI_addr_exchange_root_ctx(void)
 /* Step 2 & 3: exchange non-root contexts */
 
 /* Macros to reduce clutter, so we can focus on the ordering logics.
- * Note: they are not perfectly-wraaped, but tolearable since only used here. */
+ * Note: they are not perfectly-wrapped, but tolearable since only used here. */
```

Comment:
`s/tolearable/tolerable/`. I'd also remove the hyphen from perfectly-wrapped if no one else objects.

Reply:
Fixed.